src/connectors/hits/__tests__/connectHitsWithInsights-test.ts

```diff
@@ -7,37 +7,52 @@ jest.mock('../../../lib/utils/hits-absolute-position', () => ({
 }));
 
 describe('connectHitsWithInsights', () => {
+  const defaultInitOptions = {
+    instantSearchInstance: {
+      helper: null,
+      widgets: [],
+      insightsClient: jest.fn(),
```

Comment:
This Jest function will always be the same. It would usually better to create a function to create the default init options (`getDefaultInitOptions()`) so that we get a new spy instance every time.

However, in this case, we do not need a spy. We can go with `() => {}`.